Source/Core/VideoCommon/Present.cpp

```diff
@@ -441,21 +541,35 @@ void Presenter::UpdateDrawRectangle()
     crop_width = win_width;
   }
 
-  // ensure divisibility by 4 to make it compatible with all the video encoders
+  int int_draw_width;
+  int int_draw_height;
+
   if (g_frame_dumper->IsFrameDumping())
   {
-    draw_width = std::ceil(draw_width) - static_cast<int>(std::ceil(draw_width)) % 4;
-    draw_height = std::ceil(draw_height) - static_cast<int>(std::ceil(draw_height)) % 4;
+    // ensure divisibility by 4 to make it compatible with all the video encoders
```

Comment:
Can you change this to:

`ensure divisibility by VIDEO_ENCODER_LMC to make it compatible with all the video encoders`

it's unlikely that constant will ever change but who knows :)